data/sql/updates/pending_db_world/update-sai-ssc-trash.sql

```diff
@@ -0,0 +1,120 @@
+(21913, 0, 0, 0, 0, 0, 100, 0, 6000, 10300, 8450, 10750, 0, 0, 11, 39848, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Water Globule - In Combat - Cast Water Globule New Target'),-- fully sniffed
+(21913, 0, 1, 0, 0, 0, 100, 0, 6000, 42700, 4850, 14550, 0, 0, 11, 37871, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Water Globule - In Combat - Cast Freeze'),-- fully sniffed
+(21913, 0, 2, 0, 31, 0, 100, 512, 37871, 0, 0, 0, 0, 0, 41, 500, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Water Globule - Spell Hit Target - Despawn'),
+(21920, 0, 0, 0, 0, 0, 100, 0, 7650, 26050, 7250, 26950, 0, 0, 11, 41932, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Tidewalker Lurker - In Combat - Cast Carnivorous Bite'),-- fully sniffed
+(21964, 0, 0, 0, 0, 0, 100, 0, 15750, 15750, 47250, 51550, 0, 0, 11, 38337, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Summon Cyclone'),-- fully sniffed
+(21964, 0, 1, 0, 0, 0, 100, 0, 27900, 27900, 6050, 19750, 0, 0, 11, 38335, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Water Bolt Volley'),-- fully sniffed
+(21964, 0, 2, 0, 0, 0, 100, 0, 23050, 23050, 24250, 33250, 0, 0, 11, 38358, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Tidal Surge'),-- fully sniffed
+(21964, 0, 3, 0, 14, 0, 100, 0, 70000, 200, 15000, 15000, 0, 0, 11, 38330, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - Friendly Missing Health - Cast Healing Wave'),-- not sniffed
+(21964, 0, 4, 5, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 6, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38451, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Death - Cast Power of Caribdis'),-- N/A note: missing ability 38449 (on hp%?)
+(21965, 0, 0, 0, 0, 0, 100, 0, 10900, 10900, 10900, 14700, 0, 0, 11, 38234, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Frost Shock'),-- fully sniffed
+(21965, 0, 1, 0, 0, 0, 100, 0, 17000, 17000, 13350, 24250, 0, 0, 11, 38304, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Earthbind Totem'),-- fully sniffed
+(21965, 0, 2, 0, 0, 0, 100, 0, 15000, 15000, 30000, 30000, 0, 0, 11, 38306, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Poison Cleansing Totem'),-- not sniffed
+(21965, 0, 3, 0, 0, 0, 100, 0, 15800, 15800, 24250, 76750, 0, 0, 11, 38236, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Spitfire Totem'),-- fully sniffed
```

Comment:
Same as the other summons, not sure if these don't require the previous ones to die before the timer starts

Reply:
I will remove the smart_scripts for the boss adds from this PR and will add them to my standing PR for fathomlord where I will also delete the smart_script entries

Reply:
you're 2/2 btw 
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/83884799/baa4095d-3827-44fe-9a0b-f1d41f648264)